README.md

```diff
@@ -75,6 +75,14 @@ When multiple proxy configurations exists, precedence is taken in this form:
 2. `https_proxy` takes precedence over `.contentfulrc.json`
 2. `https_proxy` takes precedence over `http_proxy`
 
+### Proxy troubleshooting
+
+* Unable to connect? Try settings `rawProxy: true` in your `.contentfulrc.json` via:
+
+```sh
+contentful config add --raw-proxy true
```

Comment:
`--raw-proxy` is enough since this is a boolean

Reply:
No it is not, the command will complain otherwise.

```
$ ./bin/contentful.js config add --raw-proxy
/...contentful-cli/dist/cli.js:28
  if (err) throw err;
           ^
YError: Not enough arguments following: raw-proxy
...
```

Reply:
I removed the part from the code now which requires the true. WIll see in my next commit.